Repository/AbstractPhpcrRepository.php

```diff
@@ -147,6 +148,15 @@ public function add($path, $resource)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function reorder($sourcePath, $position)
```

Comment:
This means we have to add the position to the resource payload data. Front-end libraries use drag 'n drop, which uses the "placed before/after" reorder features. This means front-end libraries have to extract the position of the before/after node and pass that to the reorder.